modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -322,6 +322,16 @@ public static float Randf()
             return godot_icall_GD_randf();
         }
 
+        /// <summary>
+        /// Returns a normally-distributed pseudo-random number, using Box-Muller transform with the specified <c>mean</c> and a standard <c>deviation</c>.
+        /// This is also called Gaussian distribution.
+        /// </summary>
+        /// <returns>A random normally-distributed <see langword="float"/> number.</returns>
+        public static double Randfn(double mean, double deviation)
```

Comment:
I'd move it after `RandRange(int, int)` for consistency.

The order is a bit different in C# it seems as it does:
```
randf
randi
randf_range
randi_range
```
but that's probably not a big deal :)

Reply:
I think it's sorted alphabetically in the C# file rather than that.

Reply:
Good catch.